lib/taxonomy/publishing_api_linked_edition_parser.rb

```diff
@@ -0,0 +1,38 @@
+module Taxonomy
+  class PublishingApiLinkedEditionParser
+    attr_accessor :linked_edition
+
+    def initialize(edition_response)
+      @linked_edition = Taxon.new(edition_response.to_h.symbolize_keys)
+      @name_field = name_field
+    end
+
+    def add_expanded_links(expanded_links_response)
```

Comment:
I plan on deleting this class once I get the chance. It's copy-pasta that's only here because handling Draft taxonomies seemed outside of the scope of this PR.

I'd rather not touch it beyond extracting the class into it's own file, since it's not tested and I hope it will go away very soon.